tech/database/sqlite/about.md

```diff
@@ -24,12 +24,16 @@ $ sudo dnf install sqlite-devel sqlite-tcl sqlite-jdbc
 ## Graphical clients
 
 The `sqlite` client shipped with the basic database engine is command line (CLI)
-based. If you prefer an application with graphical user interface (GUI), install
-the `sqliteman` package:
+based. If you prefer an application with graphical user interface (GUI), you can install
+the `sqliteman` package and `sqlitebrowser`:
```

Comment:
Would you mind to elaborate at the diffirence between /usage of/ those two?

Reply:
Sqliteman is discontinued. The project is no longer developed. Last version, 1.2.0, was released in 2013.

Reply:
Then the unmaintained one should be omitted, in case they both provide the same functionallity.
We should give user an an advice, not list all possible options.

Reply:
May I remove the unmaintained 6 years old software (sqliteman) ?

Reply:
Sure thing. Let's recommend what's `best` (TM).